R/subscript-tables.R

```diff
@@ -610,7 +610,8 @@ updateQuestionTypesFromArgs <- function(dropped.dims, question.type) {
 getFallbackQuestionType <- function(question.types) {
     n.question.types <- length(question.types)
     if (n.question.types == 1L) {
-        if (question.types == "NumberMulti") return("Number")
+        if (question.types %in% c("NumberMulti", "NumberGrid")) return("Number")
+        else if (question.types %in% c("PickOneMulti", "PickAnyGrid")) return("PickAny")
```

Comment:
`else if` redundant here due to the return on line 613.

Similar code might be needed for the `n.question.types == 2` case? If two Grid variable sets (`n.question.types = 2`) are reduced to a single value (so both Q types are `NULL` after dropping) then it wont reach this block.